express-post-json/index.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+
+const app = express();
+
+const gradesArray = [];
+let nextId = 1;
+let grades = {};
```

Comment:
Try storing your grades as properties of the object data model instead of the array.